docker-compose.yml

```diff
@@ -1,30 +1,44 @@
 version: '3.1'
 services:
   humhub:
-    build: .
-    links:
-      - "db:db"
-    ports:
-      - "8080:80"
+    build:
+      context: .
+      args:
+        HUMHUB_VERSION: '1.3.11'
     environment:
-      HUMHUB_DB_USER: root
-      HUMHUB_DB_PASSWORD: root
-      HUMHUB_AUTO_INSTALL: 1
-      HUMHUB_DEBUG: 1
+      HUMHUB_DB_USER: ${HUMHUB_DB_USER}
+      HUMHUB_DB_PASSWORD: ${HUMHUB_DB_PASSWORD}
+      HUMHUB_ADMIN_USER: ${HUMHUB_ADMIN_USER}
+      HUMHUB_ADMIN_EMAIL: ${HUMHUB_ADMIN_EMAIL}
+      HUMHUB_ADMIN_PASSWORD: ${HUMHUB_ADMIN_PASSWORD}
+      HUMHUB_DB_HOST: 'db'
+      HUMHUB_DB_NAME: 'humhub'
+      HUMHUB_AUTO_INSTALL: 'true'
+      HUMHUB_DEBUG: 'true'
+      HUMHUB_PROTO: 'https'
+      HUMHUB_HOST: 'localhost'
+      HUMHUB_SUB_DIR: '/humhub'
+      PHP_MAX_EXECUTION_TIME: 321
+      PHP_UPLOAD_MAX_FILESIZE: 6M
+      PHP_POST_MAX_SIZE: 7M
+    ports:
+    - "80:80"
 #   volumes:
 #     - "${HUMHUB_VOLUME_CONFIG}:/var/www/localhost/htdocs/protected/config"
     healthcheck:
-      test: ["CMD", "curl", "-f", "http://localhost/ping"]
+      test: ["CMD", "curl", "-f", "http://localhost/humhub/"]
       interval: 30s
       timeout: 10s
       retries: 3
   db:
     image: mariadb
     environment:
-      MYSQL_ROOT_PASSWORD: root
-      MYSQL_DATABASE: humhub
-    healthcheck:
-      test: ["CMD", "mysqladmin", "ping", "--silent"]
-      interval: 30s
-      timeout: 10s
-      retries: 3
+      MYSQL_ROOT_PASSWORD: ${MARIADB_ROOT_PASSWORD}
```

Comment:
I suppose I could include a `.env` file. I usually follow this pattern just so that I can include a `docker-compose.yml` file and not expose any passwords but dummy passwords would be fine for a sample `.env` file in this case (and for the added convenience of having the password in once place but shared with the db and app, of course).